### ISSUE TITLE
Wrong Go versions in skynode and skymgr scripts

### ISSUE THREAD

**Issue by Asgaror** [0pcom/qubes-skywire#2]
[Node](https://github.com/0pcom/qubes-skywire/blob/452d16bca1a1050a5aff4c04dc618238343f0e38/skynode#L3) and the [manager]() install scripts still work with the old Go versions and weren't updated correctly.
`skydom0` downloads Go 1.10.3 and the scripts are working on 1.9.

Relace lines 2-4 and 3-5 respectively: 
```
mv ~/QubesIncoming/dom0/go1.9.linux-amd64.tar.gz /home/user/
tar xvf go1.9.linux-amd64.tar.gz
rm go1.9.linux-amd64.tar.gz
```
with
```
mv ~/QubesIncoming/dom0/go1.10.3.linux-amd64.tar.gz /home/user/
tar xvf go1.10.3.linux-amd64.tar.gz
rm go1.10.3.linux-amd64.tar.gz
```

**Comment by 0pcom** [comment]
Its fixed in the release i think, and included is go v 10.0.3

**Comment by 0pcom** [comment]
I changed it, thanks for alerting
